scripts/install_moodle.sh

```diff
@@ -883,6 +883,9 @@ EOF
    #else # mssql. TODO It's missed earlier! Complete this!
    fi
 
+   # setting the local cache file directory
+   sed -i "23 a \$CFG->localcachedir = '/tmp/moodle';" /moodle/html/moodle/config.php
```

Comment:
We've already done this (even with a ramdisk), but it didn't improve perf in any noticeable way. I think we should still have this anyway, but I don't think `/tmp/moodle` is a good location. Can we use some place like `/var/run/moodle`?